wordpress-mobile/WordPress-iOS
# Replace FLAnimatedImage library for Gifu library

Fixes #9240 

This PR swap `FLAnimatedImage` for `Gifu`, with the minimum changes possible to the rest of the project.

To test:
- Create posts with gifs as featured images.
- If you see the gifs animating in the posts list, everything is fine! 🎉
